### ISSUE TITLE
face_recognition module does not recognise any face covered with mask.

### ISSUE THREAD

**Issue by harry418** [ageitgey/face_recognition#1163]
* face_recognition version:
* Python version:
* Operating System:

### Description

Describe what you were trying to get done.
Tell us what happened, what went wrong, and what you expected to happen.
IMPORTANT: If your issue is related to a specific picture, include it so others can reproduce the issue.

### What I Did

```
Paste the command(s) you ran and the output.
If there was a crash, please include the traceback here.
```

**Comment by zenjieli** [comment]
That's because the pretrained model provided by Dlib didn't consider face with masks. But you can train your own Dlib face detection model according to http://dlib.net/train_object_detector.py.html.

**Comment by harry418** [comment]
> That's because the pretrained model provided by Dlib didn't consider face with masks. But you can train your own Dlib face detection model according to http://dlib.net/train_object_detector.py.html.

Thank you, but you can use various models:-
(1) Deep Learning based Face Detector in OpenCV
(2)  Histogram of Oriented Gradients (HOG) in Dlib
(3) NMOD Convolutional Neural Network in Dlib
you can check reference on https://github.com/AyushExel/ml4face-detection/blob/master/Final-Face%20Detection%20Notebook.ipynb
